recipes/pandas/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version="0.18.1" %}
+
+package:
+    name: pandas
+    version: {{ version }}
+
+source:
+    fn: pandas-{{ version }}.tar.gz
+    url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
+    sha256: 1f3136f69a58d490ba9f9874e67601042a11138270515dedcf6979c003bba79b
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --root=/
```

Comment:
I've been thinking about this for a while. OpenSUSE has a `spec-cleaner`\* script that does exactly this kind of thing and could be adapted here. This is beyond the linter (which they also have) and should be used with caution since it changes the recipe automatically, but it is still valuable and saves a lot of time.

\* The [`spec-cleaner`](https://github.com/openSUSE/spec-cleaner) corrects the comments, adds bolierplate comments on the patches for manual correction later, fixes the licenses, etc. There are plenty of ideas there to steal :wink: